src/Grid/grid.jl

```diff
@@ -100,27 +123,53 @@ end
 ##########################
 # Grid utility functions #
 ##########################
-@inline getdim(::AbstractGrid{dim}) where {dim} = dim
+@inline getdim(grid::AbstractGrid{dim}) where {dim} = dim
+"""
+    getcells(grid::AbstractGrid) 
+    getcells(grid::AbstractGrid, v::Union{Int,Vector{Int}} 
+    getcells(grid::AbstractGrid, set::String)
+
+Returns either all `cells::Vector{C<:AbstractCell}` of a `grid` or a subset based on an `Int`, `Vector{Int}` or `String`.
+Whereas the last option tries to call a `cellset` of the `grid`.
+"""
 @inline getcells(grid::AbstractGrid) = grid.cells
 @inline getcells(grid::AbstractGrid, v::Union{Int, Vector{Int}}) = grid.cells[v]
 @inline getcells(grid::AbstractGrid, set::String) = grid.cells[collect(grid.cellsets[set])]
+"Returns and `Int` corresponding to how many cells are in the `grid`."
```

Comment:
Describes the implementation, not the interface. Maybe something like "Returns the number of cells in the grid." or just "Number of cells in the grid." or smth like this?